src/webui/package.json

```diff
@@ -9,11 +9,12 @@
     "copy-to-clipboard": "^3.0.8",
     "echarts": "^4.1.0",
     "echarts-for-react": "^2.0.14",
-    "react": "^16.4.2",
-    "react-dom": "^16.4.2",
+    "react": "^16.7.0-alpha.2",
+    "react-dom": "^16.7.0-alpha.2",
```

Comment:
Is this upgrade necessary?  since they are alpha versions.

Reply:
I've forgot why change it. I'll test with before package.json again.

Reply:
Had changed back.

Reply:
If don't use react alpha version, will met the problem "_react.default.useContext is not a function". I haven't found a better solution for it.

Reply:
ok

Reply:
ok